index.html

```diff
@@ -0,0 +1,578 @@
+      identity
+    </aside></span>/<span style='--depth: 2' class=green>v1.0<aside>version</aside></span>
+  </code>
+</figure>
+
+The final two segments of the URL's [path](https://tools.ietf.org/html/rfc3986#section-3.3) **should** contain the spec's [default prefix](#prefix) and a [version tag](#versioning). The content of the URL up to and including the prefix—but excluding the version tag and trailing `/`—is the spec's *identity*. For the above example,
+<dl>
+  <dt markdown>`identity: "https://spec.example.com/a/b/c/exampleSpec"`</dt>
+  <dd>A global identifier for the spec. Processors can treat this as an opaque string identifying the spec (but not the version of the spec) for purposes of selecting an appropriate implementation.</dd>
+  <dt markdown>`name: "exampleSpec"`</dt>
+  <dd markdown>The spec's name, for purposes of [prefixing](#prefixing) schema elements it defines or extends.</dd>
+  <dt markdown>`version: "v1.0"`</dt>
+  <dd markdown>The tag for the [version](#versioning) of the spec used to author the document. Processors **must** select an implementation of the spec which can [satisfy](#satisfaction) the specified version.</dd>
+</dl>
+
+The version tag **must** satisfy the [BNF grammar for `‹version tag›`](#backus-naur-form-for-versions-and-version-tags).
+
+### Ignore meaningless URL components
+
+When extracting the URL's `prefix` and `version`, processors **must** ignore any url components which are not assigned a meaning. This spec assigns meaning to the final two components of the [path](https://tools.ietf.org/html/rfc3986#section-3.3). Other URL components—particularly query strings and fragments, if present—**must** be ignored for the purposes of extracting the `prefix` and `version`.
+
+<figure>
+  <figcaption>
+    <a name=spec-url-ignore-meaningless href=#spec-url-ignore-meaningless class=a-header>Ignoring meaningless parts of a URL</a>
+  </figcaption>  
+  <code class=anatomy>
+    <span class=pink style='--depth: 2'>https://example.com/<span>exampleSpec<aside>name</aside></span><aside>
+      identity
+    </aside></span>/<span style='--depth: 2' class=green>v1.0<aside>version</aside></span><span class=grey>?key=val&k2=v2#fragment<aside>ignored</aside></span>
+  </code>
+</figure>
+
+### Why is versioning in the URL, not a directive argument?
+
+The version is in the URL because when a human reader visits the URL, we would like them to be taken to the documentation for the *version of the specification in use by this document*. Many text editors will turn URLs into hyperlinks, and it's highly desirable that clicking the link takes the user to the correct version of the docs. Putting the version information in a separate argument to the `@core` directive would require this.
+
+## Prefixing
+Specifications meant to be referenced via `@core` **should** prefix any schema elements they introduce. The prefix:
```

Comment:
This seems to just be constraining the prefix in the spec (the default prefix). But I think this is also a constraint on the valid `as` arguments?

I also think this could be better structured more like

> A string is a valid prefix if it is a string of characters valid within GraphQL typenames, with no substring consisting of two underscores (`"__"`).  The name of the spec (as derived from the spec's URL) must be a valid prefix.

because the 1 and the 2 in the list are pretty different kinds of things. (Also you don't define what "match" means.)